### PR TITLE
Verified tox is working properly with Python 2.7 and 3.5 and Django 

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**
1.8, 1.9, and 1.10

@clintonb @bderusha  please review

### CODE REVIEW

File: tests/test_generate.py [hunk 1]
```diff
@@ -23,6 +23,10 @@ class TestGenerate(TestCase):
     Tests functionality of i18n/generate.py
     """
 
+    """
```
Comment (clintonb) [inline tests/test_generate.py]: This syntax is normally reserved for docstrings. Use `#`.

File: setup.py [hunk 2]
```diff
@@ -3,18 +3,39 @@
 from setuptools import setup
 
 setup(
-    name='i18n_tools',
+    name='edx-i18n-tools',
     version='0.3.2',
```
Comment (clintonb) [inline setup.py]: Use the new constant.

**Comment by Dillon-Dumesnil** [comment]
@clintonb I'll merge on your thumb and tag so it will be uploaded to PyPI

**Comment by clintonb** [comment]
Rebase on master to get rid of the merge bubble.
👍

File: pylintrc [hunk 3]
```diff
@@ -53,7 +53,8 @@ disable=
     too-many-return-statements,
     too-many-branches,
     too-many-arguments,
-    too-many-locals
+    too-many-locals,
+    missing-docstring
```
Comment (nedbat) [inline pylintrc]: Why are we ignoring missing docstring!?

File: scripts/podup.py [hunk 4]
```diff
@@ -1,41 +0,0 @@
-"""
```
Comment (nedbat) [inline scripts/podup.py]: Why did you delete this file?

File: .gitignore [hunk 5]
```diff
@@ -4,7 +4,7 @@
 *.orig
 *.DS_Store
 build
-.coverage
+.coverage*
```
Comment (nedbat) [inline .gitignore]: This ignores the .coveragerc file, which should not be ignored.

File: pylintrc (hunk 3 above)
Comment (Dillon-Dumesnil) [inline pylintrc]: @nedbat Because the quality build was failing from too many missing docstrings (and no other issues) and we decided to remove the caveat that make quality could still pass with 20 pylint errors. Seemed better to ignore the one issue that was still happening rather than let a build succeed because there are < 20 errors.

File: scripts/podup.py (hunk 4 above)
Comment (Dillon-Dumesnil) [inline scripts/podup.py]: @nedbat It was not being used anywhere

File: setup.py [hunk 6]
```diff
@@ -2,19 +2,42 @@
 
 from setuptools import setup
 
+import i18n
```
Comment (nedbat) [inline setup.py]: Importing the product code into the setup.py is fragile.  I assume this is why you had to move the import into `__call__` in `i18n/__init__.py`.  It would be better to not do this.

File: .gitignore (hunk 5 above)
Comment (Dillon-Dumesnil) [inline .gitignore]: @nedbat I'll put together a fix for that and tag you in the PR. It was added in to get rid of all of the coverage files that were being written as a part of running tox commands, but I'll update to include coveragerc

File: pylintrc (hunk 3 above)
Comment (nedbat) [inline pylintrc]: But then no one ever has to write docstrings again.  Even a diligent developer who looks at the pylint errors won't be told that they are missing a docstring.

File: setup.py (hunk 6 above)
Comment (Dillon-Dumesnil) [inline setup.py]: @nedbat The import had to be moved into __call__ because in just trying to import i18n, it was importing all of config and then failing on not being able to import yaml. I based the importing i18n into the setup file from the edx-drf-extensions repo.

Comment (nedbat) [inline setup.py]: Right, that's the kind of fragility I'm talking about.  And the next developer could add an import to `__init__.py` without understanding the need to avoid importing third-party libraries, even transitively, into there, or the installation will fail.

Comment (Dillon-Dumesnil) [inline setup.py]: @clintonb Do you have thoughts on this? I know there was a comment before on this PR to use the __version__ constant.

Comment (clintonb) [inline setup.py]: I see the value in being able to programmatically determine the package version when debugging. I also see @nedbat's point. However, if the package fails to install, the tests won't run; so, we do have protection against a developer breaking the package.

Comment (nedbat) [inline setup.py]: BTW: other techniques that don't have this fragility: https://packaging.python.org/single_source_version/
